crypto/aes256.go

```diff
@@ -0,0 +1,66 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"fmt"
+	"io"
+)
+
+// Encrypt combines secretKey and secondKey to encrypt with AES256-GCM method.
+func Encrypt(secretKey, additional, data []byte) ([]byte, error) {
```

Comment:
Ah, If these functions are merged into main branch, I'll change it when executing `approveOracleRegistration` tx.